### DISCUSSION TITLE
no subject alternative dns name matching netbeans.apache.org found [non resolu]

### DISCUSSION THREAD

**Discussion by ProgMaths** [apache/netbeans#3640] [Q&A]
Bonjour.
J'ai un pb technique mais comme je ne suis pas du tout dégourdi tant sur l'anglais que sur les manip d'ordi, je n'arrive pas à me débloquer.
J'ai installé la version 12.6 de netbeans sous windows (J'avais une version plus ancienne sous ubuntu que je n'arrivais pas à mettre à jour car, comme ubuntu, il n'arrivait pas à se connecter à internet) et j'ai fini par abandonner pour télécharger la dernière version sous windows.... mais j'ai encore ce même pb de connexion à internet... mais cette dernière version précise l'erreurs :
no subject alternative dns name matching netbeans.apache.org found.
Donc je n'arrive pas à faire mon premier push sur Github.
J'ai cherché sur le net... j'ai juste réussit à comprendre (enfin, il me semble) que c'était un problème de certificat en lien avec java, mais je ne comprends pas comment régler le pb (et sur quel forum il faudrait aller, à qui demander)...
Je ne sais pas si le fait d'avoir CyberGost d'installé (mais non actif pendant que netbeans ou ubuntu tourne) a une incidence...
bref, je suis perdu...
Dans l'espoir d'une solution. Cdt

Edit : Bonjour et merci pour votre intérêt.
Pourquoi avoir bloqué la discussion ?
Ma connexion à internet marche très bien puis que je peux poster sur le cite, c'est juste que netbeans ne la trouve pas et je ne sais pas pourquoi ... Mais cela ne m'a pas empêché d'installer un pluging disponible dans neatbeans. 
Dans l'attente de vos conseils et en vous remerciant.
Cordialement

**Comment by vieiro** [comment]
Hola @ProgMaths ,

Si no tienes conexión a Internet, no creo que puedas hacer un push a github. Creo que deberías solucionar tu problema de conectividad antes de nada.

Un saludo,
Antonio

----

Hi @ProgMaths ,

If you don't have a connection to Internet you won't be able to make a push to github. Please try to solve your Internet connection and tell us then.

Thanks,
Antonio
